### DISCUSSION TITLE
beatmap AR is always 5 when downloading very old maps

### DISCUSSION THREAD

**Discussion by hostilew** [ppy/osu#16682] [Q&A] [answered]
When downloading very old beatmaps the AR is always 5. I downloaded about 5 maps and restarted lazer and the problem persists, all maps that aren't supposed to be AR 5. tested on Cranky - The Five, Beck - Timebomb, Hyadain - Chocobo, onoken - bms-Felys and survival dance.

![osu_2022-01-28_02-25-59](https://user-images.githubusercontent.com/38239509/151513297-76f595a9-c760-47ab-b58d-1f872eff7061.jpg)

Probably affects all beatmaps uploaded before AR and OD were split?

**Comment by peppy** [comment, marked as answer]
Yeah this does ring a bell.. I believe we need to add logic to correctly transfer the value across beyond a certain `BeatmapVersion`

**Reply by smoogipoo** [reply]
https://github.com/ppy/osu/pull/16686
